src/main/java/io/github/opencubicchunks/cubicchunks/chunk/ICubeProvider.java

```diff
@@ -2,11 +2,25 @@
 
 import javax.annotation.Nullable;
 
+import io.github.opencubicchunks.cubicchunks.chunk.cube.BigCube;
 import net.minecraft.world.level.chunk.ChunkStatus;
 
 public interface ICubeProvider {
 
     @Nullable
     IBigCube getCube(int cubeX, int cubeY, int cubeZ, ChunkStatus requiredStatus, boolean load);
 
+    @Nullable
+    default BigCube getCube(int cubeX, int cubeY, int cubeZ, boolean create) {
+        return (BigCube) this.getCube(cubeX, cubeY, cubeZ, ChunkStatus.FULL, create);
+    }
+
+    @Nullable
+    default BigCube getCubeNow(int cubeX, int cubeY, int cubeZ) {
+        return this.getCube(cubeX, cubeY, cubeZ, false);
+    }
+
+    default boolean hasCube(int cubeX, int cubeY, int cubeZ) {
+        return this.getCube(cubeX, cubeY, cubeZ, ChunkStatus.FULL, false) != null;
```

Comment:
Shouldn't this use `getCubeNow` instead of `getCube`? The latter would result in the cube being loaded and thus end up returning true at all times.

Reply:
The `false` on the end means to not create a cube. This also matches the vanilla method for `hasCube`